### ISSUE TITLE
There should be a way to configure 'autoopen' on-off

### ISSUE THREAD

**Issue by stackchetan** [eugenegantz/MSelectDialogBox#7]
Currently whenever focus goes in to the input, options dropdown open by default. There should be a way to disable this feature. From accessibility point of view, user should be able to open the options by pressing enter button or clicking into the input textbox.

**Comment by eugenegantz** [comment]
please check last commits

**Comment by eugenegantz** [comment]
please check last commits
